Applications/Games/Elite:Dangerous/Steam/script.js

```diff
@@ -1,25 +1,23 @@
 include(["engines", "wine", "quick_script", "steam_script"]);
-include(["engines", "wine", "plugins", "csmt"]);
 include(["engines", "wine", "verbs", "corefonts"]);
-include(["engines", "wine", "verbs", "dotnet452"]);
-include(["engines", "wine", "verbs", "vcrun2012"]);
-include(["engines", "wine", "verbs", "quartz"]);
+include(["engines", "wine", "verbs", "dotnet45"]);
+include(["engines", "wine", "verbs", "vcrun2015"]);
+include(["engines", "wine", "verbs", "dxvk"]);
 
 var installerImplementation = {
     run: function () {
         new SteamScript()
             .name("Elite:Dangerous")
             .editor("Frontier Developments")
             .author("ImperatorS79")
-            .wineVersion(LATEST_STAGING_VERSION)
-            .wineDistribution("staging")
+            .wineVersion(LATEST_STABLE_VERSION)
+            .wineDistribution("upstream")
```

Comment:
You can remove this as upstream is the default distribution. Unless it is really important to use upstream and you need to guarantee that it will be used.